unstructured/embed/mixedbreadai.py

```diff
@@ -0,0 +1,211 @@
+import os
+from dataclasses import dataclass, field
+from typing import TYPE_CHECKING, List, Optional
+
+import numpy as np
+
+from unstructured.documents.elements import Element
+from unstructured.embed.interfaces import BaseEmbeddingEncoder, EmbeddingConfig
+from unstructured.ingest.error import EmbeddingEncoderConnectionError
+from unstructured.utils import requires_dependencies
+
+USER_AGENT = "mixedbread-ai@unstructured"
```

Comment:
@mixedbread-ai/unstructured

Reply:
then we need to change those in haystack and langchain integrations as well.